config/config.go

```diff
@@ -158,6 +158,8 @@ type Config struct {
 	EnableGlobalIndex bool `toml:"enable-global-index" json:"enable-global-index"`
 	// DeprecateIntegerDisplayWidth indicates whether deprecating the max display length for integer.
 	DeprecateIntegerDisplayWidth bool `toml:"deprecate-integer-display-length" json:"deprecate-integer-display-length"`
+	// EnableEnumLengthLimit enables the enum element length limit.
+	EnableEnumLengthLimit bool `toml:"enable-enum-length-limit" json:"enable-enum-length-limit"`
```

Comment:
how about using a global variable instead of a config item?

Reply:
MYSQL8.0 don't provide a var to control enum/set length. Is it ok to add a ScopeGlobal var like "tidb_enable_enum_length_limit" ?
https://dev.mysql.com/doc/refman/8.0/en/server-system-variable-reference.html